SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -0,0 +1,530 @@
+							+ SVerticalBox::Slot()
+							.AutoHeight()
+							.Padding(2.0f)
+							[
+								SNew(SHorizontalBox)
+								+ SHorizontalBox::Slot()
+								.FillWidth(1.0f)
+								[
+									SNew(STextBlock)
+									.Text(FText::FromString(FString(TEXT("Deployment Name"))))
+									.ToolTipText(FText::FromString(FString(TEXT("The name of the deployment."))))
+								]
+								+ SHorizontalBox::Slot()
+								.FillWidth(1.0f)
+								[
+									SNew(SEditableTextBox)
+									.Text(FText::FromString(SpatialGDKCloudLauncherSettings->GetPrimaryDeploymentName()))
+									.ToolTipText(FText::FromString(FString(TEXT("The name of the deployment."))))
+									.OnTextCommitted(this, &SSpatialGDKSimulatedPlayerDeployment::OnPrimaryDeploymentNameCommited)
+									.OnTextChanged(this, &SSpatialGDKSimulatedPlayerDeployment::OnPrimaryDeploymentNameCommited, ETextCommit::Default)
+								]
+							]
+							// Snapshot File + File Picker
+							+ SVerticalBox::Slot()
+							.AutoHeight()
+							.Padding(2.0f)
+							[
+								SNew(SHorizontalBox)
+								+ SHorizontalBox::Slot()
+								.FillWidth(1.0f)
+								[
+									SNew(STextBlock)
+									.Text(FText::FromString(FString(TEXT("Snapshot File"))))
+									.ToolTipText(FText::FromString(FString(TEXT("The relative path to the snapshot file."))))
+								]
+								+ SHorizontalBox::Slot()
+								.FillWidth(1.0f)
+								[
+									SNew(SFilePathPicker)
+									.BrowseButtonImage(FEditorStyle::GetBrush("PropertyWindow.Button_Ellipsis"))
+									.BrowseButtonStyle(FEditorStyle::Get(), "HoverHintOnly")
+									.BrowseButtonToolTip(FText::FromString(FString(TEXT("Path to the snapshot file"))))
+									.BrowseDirectory(SpatialGDKSettings->GetSpatialOSSnapshotFolderPath())
+									.BrowseTitle(FText::FromString(FString(TEXT("File picker..."))))
+									.FilePath_UObject(SpatialGDKCloudLauncherSettings, &USpatialGDKEditorCloudLauncherSettings::GetSnapshotPath)
+									.FileTypeFilter(TEXT("Snapshot files (*.snapshot)|*.snapshot"))
+									.OnPathPicked(this, &SSpatialGDKSimulatedPlayerDeployment::OnSnapshotPathPicked)
+								]
+							]
+							// Primary Launch Config + File Picker
+							+ SVerticalBox::Slot()
+							.AutoHeight()
+							.Padding(2.0f)
+							[
+								SNew(SHorizontalBox)
+								+ SHorizontalBox::Slot()
+								.FillWidth(1.0f)
+								[
+									SNew(STextBlock)
+									.Text(FText::FromString(FString(TEXT("Launch Config File"))))
+									.ToolTipText(FText::FromString(FString(TEXT("The relative path to the launch configuration file."))))
+								]
+								+ SHorizontalBox::Slot()
+								.FillWidth(1.0f)
+								[
+									SNew(SFilePathPicker)
+									.BrowseButtonImage(FEditorStyle::GetBrush("PropertyWindow.Button_Ellipsis"))
+									.BrowseButtonStyle(FEditorStyle::Get(), "HoverHintOnly")
+									.BrowseButtonToolTip(FText::FromString(FString(TEXT("Path to the primary launch configuration"))))
```

Comment:
What does "primary" mean?

Reply:
Discussed offline - will remove the "primary" from the tooltip.